SPECS/gh/gh.spec

```diff
@@ -25,10 +26,12 @@ Requires:       git
 GitHub official command line tool.
 
 %prep
-%autosetup -p1 -n cli-%{version}
+# Don't patch during setup to apply vendor package 'go-gh' patch CVE-2024-53859.patch
```

Comment:
Please remove the CVE number and package name from comment. It's applicable for other CVEs and packages as well (For eg. I am fixed one in this PR: https://github.com/microsoft/azurelinux/pull/11767)